src/main/scala/com/actian/spark_vector/colbuffer/util/BigIntegerConversion.scala

```diff
@@ -22,29 +22,45 @@ import java.nio.ByteBuffer
 
 /** Helper functions and constants for `BigInteger` conversions. */
 object BigIntegerConversion {
+  val bigIntArray = Array.fill[Byte](LongLongSize)(0:Byte) /** Keep it in big-endian */
```

Comment:
Add a note to this object's doc that warns users that they should not call methods of this object concurrently.